lib/nimble_template/projects/project.ex

```diff
@@ -1,10 +1,10 @@
 defmodule NimbleTemplate.Projects.Project do
   @moduledoc false
 
-  @alpine_version "3.14.6"
-  @elixir_version "1.14.0"
-  @erlang_version "25.0.4"
-  @node_asdf_version "16.15.0"
+  @alpine_version "3.17.2"
```

Comment:
The latest alpine version on hex docker image is 3.17.0 FYI @Nihisil :) 

https://hub.docker.com/layers/hexpm/elixir/1.14.3-erlang-25.2.3-alpine-3.17.0/images/sha256-55a63447e18a47f72b811f89dea73a3b9307e91fa6b01139e0022d0d9ae2a8d9?context=explore